statsmodels/tsa/statespace/tests/test_sarimax.py

```diff
@@ -0,0 +1,1149 @@
+"""
+Tests for SARIMAX models
+
+Author: Chad Fulton
+License: Simplified-BSD
+"""
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+import pandas as pd
+import os
+
+import warnings
+from statsmodels.tsa.statespace import sarimax, tools
+from .results import results_sarimax
+from statsmodels.tools import add_constant
+from numpy.testing import assert_almost_equal, assert_raises, assert_allclose
+from nose.exc import SkipTest
+
+current_path = os.path.dirname(os.path.abspath(__file__))
+
+coverage_path = 'results' + os.sep + 'results_sarimax_coverage.csv'
+coverage_results = pd.read_csv(current_path + os.sep + coverage_path)
+
+
+class SARIMAXStataTests(sarimax.SARIMAX):
+    def test_loglike(self):
+        assert_almost_equal(
+            self.result.llf,
+            self.true['loglike'], 4
+        )
+
+    def test_aic(self):
+        assert_almost_equal(
+            self.result.aic,
+            self.true['aic'], 3
+        )
+
+    def test_bic(self):
+        assert_almost_equal(
+            self.result.bic,
+            self.true['bic'], 3
+        )
+
+    def test_hqic(self):
+        hqic = (
+            -2*self.result.llf +
+            2*np.log(np.log(self.result.nobs)) *
+            self.result.params.shape[0]
+        )
+        assert_almost_equal(
+            self.result.hqic,
+            hqic, 3
+        )
+
+    # def test_bse(self):
```

Comment:
just for information: why is this commented out?   maybe add comment to code about it.